gui/mainFrame.py

```diff
@@ -405,10 +406,22 @@ def goWiki(self, event):
     def goForums(self, event):
         webbrowser.open('https://forums.eveonline.com/default.aspx?g=posts&t=466425')
 
+    def loadDatabaseDefaults(self, event):
+        # Import default database values
+        importDBDefaults = loadDefaultDatabaseValues.defaultDatabaseValues()
+        # Import values that must exist otherwise Pyfa breaks
+        importDBDefaults.importRequiredDefaults()
+        # Import default values for damage profiles
+        importDBDefaults.importDamageProfileDefaults()
+        # Import default values for target resist profiles
+        importDBDefaults.importResistProfileDefaults()
+
```

Comment:
Here, you don't need to initialize the class. After implementing the change to the import statement, simply do this:

```
def loadDatabaseDefaults(self, event):
    # Since these are class methods, you can call them directly from the class rather than initialize the class to another variable
    DefaultDatabaseValues.importRequiredDefaults()
    DefaultDatabaseValues.importDamageProfileDefaults()
    DefaultDatabaseValues.importResistProfileDefaults()
```